tests/roles/run_tests/tasks/generate_templates.yml

```diff
@@ -18,9 +18,11 @@
       mode: '0755'
 
   - name: Deploy clusterctl variables to clusterctl config
-    template:
-      src: clusterctl-vars.yaml
-      dest: "{{ HOME }}/.cluster-api/clusterctl.yaml"
+    ansible.builtin.blockinfile:
+      block: "{{lookup('ansible.builtin.template', '{{ CRS_PATH }}/clusterctl-vars.yaml')}}"
```

Comment:
can you explain what we are trying to do here exactly?

Reply:
[Append content in clusterctl.yaml instead of overwriting.](https://github.com/metal3-io/metal3-dev-env/pull/1064/commits/1b30849afd07171c6870700917344b061edaf297)

Reply:
I was also a bit wondering about this first. Previously we used ansible `template`, but this overwrites the file completely. This will instead render the template and insert it as a "block" in the file.

Reply:
ah yes, what we discussed offline, got it.

Reply:
CI runs are using the specific version and happy!
/approve